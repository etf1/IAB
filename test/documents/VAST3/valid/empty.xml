<!-- Most basic valid VAST document -->
<VAST version="3.0"></VAST>
