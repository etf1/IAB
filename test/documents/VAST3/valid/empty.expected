{
  "version": "3.0",
  "ads": []
}
